src/components/CatalogOverlay/CatalogPlotComponent/CatalogPlotComponent.tsx

```diff
@@ -702,13 +813,53 @@ export class CatalogPlotComponent extends React.Component<WidgetProps> {
             dragmode: widgetStore.dragmode
         };
 
+        if (widgetStore.showFittingResult) {
+            const fitting = widgetStore.fitting;
+            const minMaxX = widgetStore.minMaxX;
+            layout.shapes = [
+                {
+                    type: "line",
+                    layer: "above",
+                    x0: minMaxX.minVal,
+                    y0: fitting.intercept + fitting.slope * minMaxX.minVal,
+                    x1: minMaxX.maxVal,
+                    y1: fitting.intercept + fitting.slope * minMaxX.maxVal,
+                    line: {
+                        color: Colors.GREEN2,
+                        width: 5
```

Comment:
change to `2.5 * ratio` so that the line width looks the same on a devicePixelRatio = 1 screen
![image](https://user-images.githubusercontent.com/43841102/142820553-9773b6d5-2570-4e68-befe-d4fd6a715ff6.png)